dist/webpack/generate-lazy-controller.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+'use strict';
+/**
+ *
+ * @param {String} controllerPath The importable path to the controller
+ * @param {Number} indentationSpaces Amount each line should be indented
+ */
+
+module.exports = function generateLazyController(controllerPath, indentationSpaces) {
```

Comment:
We should clean this up. Some dist files *are* needed because they're read at runtime in the browser - specifically the `src/index.js` file (which the user imports directly from their code and uses. But other things - like these loaders - are only executed at build time and do not need a dist.

Reply:
even for the necessary ones, I don't think they need to be committed, as the npm ecosystem relies on a separate publishing step and so the VCS content and the package content can be different.

Reply:
You're right about that too. None of them should be committed. However, we would need to make sure there was a more sophisticated build+publish functionality. The dist files are also committed in symfony/ux for the same reason: ease of distribution (because there is no automated build/release procedure).

Reply:
symfony/ux packages are not distributed through npm but through composer.

Reply:
npm already has a built-in automation: it will run the `prepublishOnly` script before building the archive for publication.
you only need to update the package.json to make that `prepublishOnly` script run `npm run build` and that's automated.

Reply:
This repo could definitely not include the dist files at all indeed. I added them without too much thoughts after having added them in symfony/ux.

Reply:
let's fix that in another PR